components/admin/taskSteps/quizStep.tsx

```diff
@@ -0,0 +1,318 @@
+                <TextInput
+                  onChange={(e) => handleQuestionChange(e, questionIndex)}
+                  value={eachQuestion.question}
+                  name="question"
+                  label={`Describe your question`}
+                  placeholder={`Question ${questionIndex + 1}`}
+                />
+                <Typography type={TEXT_TYPE.BODY_MICRO} color="textGray">
+                  Write the question&apos;s description.
+                </Typography>
+              </div>
+              <div className="flex flex-col w-full gap-2 px-4 py-3 rounded-xl justify-center items-center border-[1px] border-[#f4faff4d]">
+                <div className="pt-4 py-6">
+                  <Typography type={TEXT_TYPE.H4} color="secondary">
+                    Set Options
+                  </Typography>
+                </div>
+                <div className="flex gap-4 flex-col w-full">
+                  {eachQuestion.options.map((option, optionIndex) => (
+                    <div
+                      key={"option-" + optionIndex}
+                      className="flex flex-row justify-between w-full items-center"
+                    >
+                      <div className="flex flex-col w-full gap-1">
+                        <label htmlFor={"option"}>
+                          Option {optionIndex + 1}
+                        </label>
+                        <div className="flex flex-row items-center">
+                          <div className="w-full">
+                            <input
+                              name={"option"}
+                              value={option}
+                              onChange={(e) =>
+                                handleOptionChange(
+                                  e,
+                                  optionIndex,
+                                  questionIndex
+                                )
+                              }
+                              placeholder={`Option ${optionIndex + 1}`}
+                              className={`${styles.input} w-full`}
+                              type={"text"}
+                            />
+                          </div>
+                        </div>
+                      </div>
+                    </div>
+                  ))}
+                </div>
+              </div>
+
+              <div className="flex flex-col gap-1">
+                <Dropdown
+                  value={
+                    step.data.questions[questionIndex].correct_answers[0] ?? 1
+                  }
+                  backgroundColor="#101012"
+                  textColor="#fff"
+                  handleChange={(event: SelectChangeEvent) => {
+                    handleCorrectAnswer(
+                      parseInt(event.target.value),
+                      questionIndex
+                    );
+                  }}
+                  options={["1", "2", "3", "4"].map((eachItem) => {
```

Comment:
I don't understand this code, why don't you direct give

```js
const options = [{value: "1", label: "1"}...];
```

Reply:
done